class-unlist-posts.php

```diff
@@ -149,9 +150,35 @@ public function hide_post_from_searchengines() {
 
 			$hidden_posts = get_option( 'unlist_posts', array() );
 
+			// wp_no_robots is deprecated since WP 5.7.
+			if ( function_exists( 'wp_robots_no_robots' ) ) {
```

Comment:
Add this snippet at the top of this method.

Reply:
Updated